content/docs/typechecking-with-proptypes.md

```diff
@@ -1,18 +1,18 @@
 ---
 id: typechecking-with-proptypes
-title: Typechecking With PropTypes
+title: Verificación de tipos con PropTypes
 permalink: docs/typechecking-with-proptypes.html
 redirect_from:
   - "docs/react-api.html#typechecking-with-proptypes"
 ---
 
-> Note:
+> Nota:
 >
-> `React.PropTypes` has moved into a different package since React v15.5. Please use [the `prop-types` library instead](https://www.npmjs.com/package/prop-types).
+> `React.PropTypes` se movió a un paquete diferente desde React v15.5. Por favor use [en su lugar la libreria `prop-types`](https://www.npmjs.com/package/prop-types).
```

Comment:
Hi @jnosornov 👍 please chance this sentence to: 
Por favor usa [en su lugar la biblioteca `prop-types`](https://www.npmjs.com/package/prop-types).
:)

Reply:
alright I will modify it